Java/Operating System/banker.java

```diff
@@ -0,0 +1,338 @@
+import java.util.Scanner;
+
+class algorithm //class for calculation and display
```

Comment:
Comments don't have to be very layman. We all understand what work does a class do.